mantis-publish/mantis-publish-core/build.gradle

```diff
@@ -17,7 +17,7 @@ ext {
     archaiusVersion = 'latest.release'
     versionMockito = '2.18.+'
     mqlVersion = '3.2.2'
-    spectatorVersion = '0.134.0'
+    spectatorVersion = '1.6.10'
```

Comment:
just wondering if we need to actually declare spectator as "API" in this module or can it be compileOnly to let application resolve it.

Reply:
This would become a breaking change if we do that right? if an application doesn't have explicit dependency on spectator api.

Reply:
that's true. I just assume all internal applications have a dependency on spectator already.